.github/workflows/codeql-analysis.yml

```diff
@@ -0,0 +1,54 @@
+name: "CodeQL"
```

Comment:
just for my information: how CodeQL applies here? AFAIK javascript is only used for the tests